ocp_resources/network_attachment_definition.py

```diff
@@ -197,3 +197,42 @@ def to_dict(self):
     @property
     def resource_name(self):
         return f"ovs-cni.network.kubevirt.io/{self.bridge_name}"
+
+
+class FlatL2NetworkAttachmentDefinition(NetworkAttachmentDefinition):
```

Comment:
Is this resource is standalone resource? or extended NetworkAttachmentDefinition ?

Reply:
It is a NetworkAttachmentDefinition.